common/mathlib.h

```diff
@@ -27,7 +27,7 @@ typedef vec_t vec4_t[4];	// x,y,z,w
 
 struct mplane_s;
 
-extern vec3_t vec3_origin;
+extern vec3_t vec3_origin_c;
```

Comment:
Alfred sdk uses some definitions as a workaround I think. But this is really messed up and I did not get the compiler happy without changing it to this.